python/test/unit/mesh/test_higher_order_mesh.py

```diff
@@ -22,6 +19,11 @@
 from ufl import dx
 
 
+scipy = pytest.importorskip("scipy")
+sympy = pytest.importorskip("sympy")
```

Comment:
FIAT depends on SymPy, so I'm surprised that this is needed.

Reply:
Indeed, it's not really needed for SymPy.